operator/v1/types_network.go

```diff
@@ -65,6 +65,11 @@ type NetworkSpec struct {
 	// +optional
 	DeployKubeProxy *bool `json:"deployKubeProxy,omitempty"`
 
+	// enableNetworkDiagnostics specifies whether or not periodic
+	// network connectivity checks should be enabled or not. If unset, this
+	// property defaults to 'true' and network diagnostics is enabled.
```

Comment:
perhaps setting a default for this would be useful?

e.g.
``
	// +kubebuilder:default=false
``
(or whatever the default is)

This would indicate to folks what to expect if this is left unset.

Reply:
done